pkg/scraper/client/resource/decode_test.go

```diff
@@ -241,3 +242,27 @@ container_start_time_seconds{container="metrics-server",namespace="kubernetes-da
 		})
 	}
 }
+
+func Fuzz_decodeBatch(f *testing.F) {
```

Comment:
I think it would be worth to seed this fuzz tests. Basically provide an values that we always want to test, in this case values that trigger bug we fixed before. This can be done by calling `f.Add(...)`

Reply:
Yeah, Thank you for reminding, I think it is necessary. I have updated.